tests/lit/single/conversion/real_trunc_integer.st

```diff
@@ -0,0 +1,19 @@
+// RUN: (%COMPILE %s && %RUN) | %CHECK %s
+FUNCTION main
+    printf('%d$N', REAL_TRUNC_SINT(4.9));  // CHECK: 4
```

Comment:
Missing `printf('%d$N', LREAL_TRUNC_REAL(4.9));  // CHECK: 4`; there might be a bug here because the `CHECK` expects some random big number like `-2124880280`